src/Mvc/HttpCodeConvention.cs

```diff
@@ -50,6 +50,16 @@ public static class HttpCodeConvention
 	/// </summary>
 	public static string ConflictProblemDetailsTitle { get; set; } = "Conflict with the current state";
 
+	/// <summary>
+	///		The HTTP code to return when failed due to a conflict with the current state of the target resource (<see cref="DomainOperationStatus.Conflict"/> status). The default value: 409
+	/// </summary>
+	public static int ContentTooLargeHttpCode { get; set; } = 413;
+	/// <summary>
+	///		The title in the returned JSON accompanying the <see cref="ContentTooLargeHttpCode"/> response (ContentTooLarge)
+	///		The default value: "Conflict with the current state of the target resource"
+	/// </summary>
+	public static string ContentTooLargeProblemDetailsTitle { get; set; } = "Client request is too large";
```

Comment:
The default value and the comment need to be changed to `The requested entity is larger than limits defined by server` as per [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413).